addons/l10n_id/models/account_move.py

```diff
@@ -0,0 +1,36 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+from odoo.tools.misc import formatLang
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def _compute_tax_totals(self):
+        """ OVERRIDE
+
+        For invoices based on ID company as of January 2025, there is a separate tax base computation for nun-luxury goods.
```

Comment:
```suggestion
        For invoices based on ID company as of January 2025, there is a separate tax base computation for non-luxury goods.
```
I wonder what nun luxury goods would represent.